PostFreeSurfer/scripts/acpc_dc_to_template

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
```

Comment:
Another script that shouldn't have been committed?

Reply:
fixex

Reply:
Did you intend to remove this file from the PR?  You have to commit the fact of its deletion (`git rm` is one way to stage a deletion for commit) and then push that commit.  Same with `convert_tp_to_template_xfm`.

Reply:
Ok.